apps/core/mutations.py

```diff
@@ -164,9 +164,37 @@ def mutate(root, info, id):
         return DeleteTableFromWorkspace(result=cloned_table, errors=None, ok=True)
 
 
+class RenameTable(graphene.Mutation):
+    class Arguments:
+        id = graphene.ID(required=True)
+        name = graphene.String(required=True)
+
+    errors = graphene.List(graphene.NonNull(CustomErrorType))
+    ok = graphene.Boolean()
+    result = graphene.Field(TableType)
+
+    @staticmethod
+    def mutate(root, info, id, name):
+        try:
```

Comment:
This logic(`try` `except`) is repeated a lot of times. Maybe we should think about abstracting this out, perhaps via decorators. Not urgent, but something to think about.

Reply:
Sure